mover-rclone/README.md

```diff
@@ -0,0 +1,47 @@
+# Rclone-based data mover
+
+## Getting started
+
+This directory contains the code for the data mover that uses rclone to
+transfer data between the source and destination sites using volumesnapshot
+feature.
+
+AWS S3 is used as an intermediate storage for high fan-out data replication.
+
+Until there's an operator to deploy this container, the following steps can be
+used for experimentation.
+
+### Prerequiste
+
+Create secret to access AWS S3 account by replacing `<access-key-id>` and
+`<secret-access-key>` in `rclone.conf`
```

Comment:
nit: also, `region` and `location-constraint` seem to be required, so should this be mentioned here?
And, is it obvious the path to _rclone.conf_?